tools/install.sh

```diff
@@ -263,10 +269,29 @@ elif [ "$LINUX_VERSION" == "BackBox" ] || \
 	 [ "$LINUX_VERSION" == "Ubuntu"  ]; then
 	apt-get install -y libfreetype6-dev python-dev python-pip
 	if [ -z "$KING_PHISHER_SKIP_CLIENT" ]; then
-		apt-get install -y gir1.2-gtk-3.0 gir1.2-gtksource-3.0 gir1.2-vte-2.90 \
+		apt-get install -y gir1.2-gtk-3.0 gir1.2-gtksource-3.0 \
+			gir1.2-webkit-3.0 python-cairo libgeos++-dev \
+			libgtk-3-dev libpq-dev python-gi python-gi-cairo libpq-dev \
+			python-gobject python-gobject-dev python-paramiko pkg-config
+		if [ $? -eq 0 ]; then
+			echo "Successfully installed dependancies"
+		else
+			echo -e "\nFailed to install the following dependencies with apt-get: \n\n \
+			gir1.2-gtk-3.0 gir1.2-gtksource-3.0 gir1.2-vte-2.90 \
 			gir1.2-webkit-3.0 python-cairo libgeos++-dev \
 			libgtk-3-dev libpq-dev python-gi python-gi-cairo \
-			python-gobject python-gobject-dev python-paramiko
+			python-gobject python-gobject-dev python-paramiko pkg-config \
+			\n\nPlease correct issues and try again"
+			exit
+		fi
+
+		apt-cache search gir1.2-vte-2.91
+		if [ $? -eq 0 ]; then
+			apt-get install gir1.2-vte-2.91
+		else
+			apt-get install gir1.2-vte-2.90
+		fi
+
```

Comment:
This should check the exit status of the last command and complain that `gir1.2-vte-2.xx` couldn't be installed. I think it would be fine if the versions were combined for this check.